src/Compilers/CSharp/Test/Semantic/Semantics/NullableReferenceTypesTests.cs

```diff
@@ -31370,16 +31695,12 @@ void M()
     {
         P.ToString();
         P = null;
-        P.ToString(); // 1
+        P.ToString();
```

Comment:
glad that we have symmetry between `[NotNull] string? P` and `[MaybeNull] string P`